backend/models/service_model.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import BaseModel, Field
+from enum import Enum
+from typing import List
+from datetime import datetime
+from typing import Optional
+from .enum_for_models import ProgramTypeEnum
+
+
+class Service(BaseModel):
+    id: int | None = None
+    created_at: datetime | None = None
+    name: str
+    status: str
+    summary: str
```

Comment:
This is good, but I think for consistency sake, we should either stick with defining as `str` or `Field(..., max_length=150, description="The name of the service")`. Similar to how Resource was defined.